transforms/language/html2parquet/notebooks/html2parquet.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c4f9c952-cb3b-40f1-bfb5-00d9a43a5715",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "!pip install data-prep-toolkit==0.2.2.dev2\n",
+    "!pip install 'data-prep-toolkit-transforms[html2parquet]'\n",
+    "!pip install pandas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "76469c87-cfd0-4acd-b57f-36edc52018ff",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "11:38:33 INFO - html2parquet parameters are : {'output_format': <html2parquet_output_format.MARKDOWN: 'markdown'>, 'favor_precision': <html2parquet_favor_precision.TRUE: 'True'>, 'favor_recall': <html2parquet_favor_recall.TRUE: 'True'>}\n",
+      "11:38:33 INFO - pipeline id pipeline_id\n",
+      "11:38:33 INFO - code location None\n",
+      "11:38:33 INFO - data factory data_ is using local data access: input_folder - input output_folder - output\n",
+      "11:38:33 INFO - data factory data_ max_files -1, n_sample -1\n",
+      "11:38:33 INFO - data factory data_ Not using data sets, checkpointing False, max files -1, random samples -1, files to use ['.html'], files to checkpoint ['.parquet']\n",
+      "11:38:33 INFO - orchestrator html2parquet started at 2024-11-18 11:38:33\n",
+      "11:38:33 INFO - Number of files is 1, source profile {'max_file_size': 0.2035503387451172, 'min_file_size': 0.2035503387451172, 'total_file_size': 0.2035503387451172}\n",
+      "11:38:33 INFO - Completed 1 files (100.0%) in 0.003 min\n",
+      "11:38:33 INFO - Done processing 1 files, waiting for flush() completion.\n",
+      "11:38:33 INFO - done flushing in 0.0 sec\n",
+      "11:38:33 INFO - Completed execution in 0.003 min, execution result 0\n"
+     ]
+    }
+   ],
+   "source": [
+    "from dpk_html2parquet.transform_python import Html2ParquetRuntime\n",
+    "x=Html2ParquetRuntime(input_folder= \"input\", \n",
+    "               output_folder= \"output\", \n",
+    "               data_files_to_use=['.html']).transform()"
```

Comment:
I know input file is just one html file but ['.zip', '.html'] in case user wants to test html zip in the notebook.

Reply:
or add a comment something like: '.zip' in case your input file is zip of html files.